python/py-lxml/Portfile

```diff
@@ -39,6 +40,11 @@ if {${name} ne ${subport}} {
                     port:libxml2 \
                     port:libxslt
 
+    if {${python.version} > 311} {
```

Comment:
why is this only needed for Python 3.12?

Reply:
I do not get any failures with earlier versions. Perhaps we could use it for `> 310` instead, since Python 3.11 itself needs C11. But earlier ones do not have to be included.

Reply:
is there any reason to try and build it ever with these old compilers? In other words, can we just do this unconditionally?

Reply:
@reneeotten I am fine with that, but I am not sure if there is any. (I would have happily avoid Xcode gcc completely aside of using it to bootstrap gcc10, to be honest.)

I will make it unconditional.

Reply:
@reneeotten Changed it to an unconditional blacklist.